paddle/phi/kernels/onednn/transpose_kernel.cc

```diff
@@ -86,31 +86,69 @@ void TransposeKernel(const Context& dev_ctx,
 
   auto x_vec_dims = vectorize(x.dims());
   auto x_type = funcs::ToOneDNNDataType(x.dtype());
+
+  dnnl::primitive_attr attrs;
+  static constexpr int32_t mask = 0;
+  const auto quantization_scale =
+      dev_ctx.HasDnnAttr("scale")
+          ? PADDLE_GET_CONST(float, dev_ctx.GetDnnAttr("scale"))
+          : 1.0f;
+  const auto quantization_shift =
+      dev_ctx.HasDnnAttr("shift")
+          ? PADDLE_GET_CONST(float, dev_ctx.GetDnnAttr("shift"))
+          : 0.0f;
+  const auto output_data_type =
+      dev_ctx.HasDnnAttr("output_data_type")
+          ? PADDLE_GET_CONST(std::string,
+                             dev_ctx.GetDnnAttr("output_data_type"))
+          : "";
+  const bool with_scale = quantization_scale != 1.0f;
+  const bool with_shift = quantization_shift != 0.0f;
+
+  if (with_scale) {
+    attrs.set_output_scales(mask, {quantization_scale});
+  }
+
+  if (with_shift) {
+    auto dst = output_data_type == "fp32" ? DNNL_ARG_SRC : DNNL_ARG_DST;
+    attrs.set_zero_points(
+        dst, mask, {static_cast<int32_t>(quantization_shift)});
+  }
+
+  DataType out_dtype;
+  if (output_data_type == "bf16") {
+    out_dtype = DataType::BFLOAT16;
```

Comment:
I think we have no test for this case here if we have performed a fuse and the fuse has set bf16

Reply:
@sfraczek So what would you suugests?